src/layouts/Media/components/MediaBreadcrumb.tsx

```diff
@@ -45,21 +48,33 @@ export const MediaBreadcrumbs = (): JSX.Element => {
           ]
         )
         .map(({ name, url }, idx) => {
-          const hasModifier = idx === directories.length - 1
+          if (
+            directories.length > 3 &&
+            idx > 1 &&
+            idx < directories.length - 2
+          ) {
```

Comment:
could we add a comment here + use the `MAX_DIRECTORIES_DEPTH` constant (iirc) defined earlier? this isn't very clear at first glance

Reply:
could i also ask why we're removing the breadcrumb here? (maybe a design decision?)

Reply:
Updated to `MAX_MEDIA_LEVELS`! Also yeah the removal is so that we can truncate the intermediate levels (although it is very unlikely to occur once we start enforcing that they can only have max 3 levels).